File: java/src/main/java/com/xia/AClass.java
package com.xia;

/**
 * Created by xia on 16/7/18.
 */
public class AClass {

    IB b = null;

    public AClass() {

    }

    public String doMethod() {
        return b.doMethod();
    }

    public void setB(IB b) {

        this.b = b;
    }


}

File: js/app/AClass.js
/**
 * Created by xia on 16/7/21.
 */

function AClass() {
    var bClass

}
AClass.prototype = {
    setBClass: function setBClass(obj) {
        bClass = obj
    },
    doMethod: function (msg) {
        console.log('bClass',bClass)

        var ret = bClass.doMethod('someText')

        console.log('bClass return:' + ret)
    }
}

module.exports = AClass;
